hibike/hibike_process.py

```diff
@@ -104,6 +107,7 @@ def device_read_thread(index, ser, instructionQueue, errorQueue, stateQueue):
             if uid is not None:
                 params_and_values = hm.parse_device_data(packet, hm.uid_to_device_id(uid))
                 stateQueue.put(("device_values", [uid, params_and_values]))
+                instructionQueue.put(("heartResp", [uid]))
```

Comment:
I know that this code was implemented this way in order to get a MVP up quickly.  We understood that there'd be a big overhead when we did it this way.

Given the latency testing that yizhe is doing now, I'd advocate moving into a model where Smart Devices send heart beat requests ASAP.